### PR TITLE
Create Webinars Pages “workshop-gitops-with-pulumi-and-gitlab/index-md”

### PR DESCRIPTION
**Pull request by isaac-pulumi**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+meta_desc: "Learn how to implement GitOps for your cloud infrastructure using
+  Pulumi and GitLab. "
+featured: false
+pre_recorded: false
+pulumi_tv: false
+preview_image: /images/webinar/gitlab-demo.png
+unlisted: false
+gated: true
+type: web
+external: true
```
Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md:12]:
```suggestion
external: false
```

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+title: "Workshop: GitOps with Pulumi and GitLab"
+meta_desc: "Learn how to implement GitOps for your cloud infrastructure using
+  Pulumi and GitLab. "
+featured: false
+pre_recorded: false
+pulumi_tv: false
+preview_image: /images/webinar/gitlab-demo.png
+unlisted: false
+gated: true
+type: web
```
Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md:11]:
```suggestion
type: webinars
```

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  duration: 02:00:00
+  title: "Workshop: GitOps with Pulumi and GitLab"
+  description: "Deploy and update your infrastructure the same way you deliver the
+    rest of your applications: with  code delivered through your CI/CD pipeline.
+    Learn how Pulumi’s modern infrastructure as code platform helps your team
+    deliver features faster by helping you to define resources on any cloud
+    using your favorite languages."
+form:
+  gotowebinar_key: "6999643165114589707"
+---
```
Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md:38]:
```suggestion
    hubspot_form_id: "ebc6064d-756c-4d04-ac63-31ad16780ab5"
---
```

Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md:38]:
```suggestion
    hubspot_form_id: "ebc6064d-756c-4d04-ac63-31ad16780ab5"
---
```

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  duration: 02:00:00
+  title: "Workshop: GitOps with Pulumi and GitLab"
+  description: "Deploy and update your infrastructure the same way you deliver the
+    rest of your applications: with  code delivered through your CI/CD pipeline.
+    Learn how Pulumi’s modern infrastructure as code platform helps your team
+    deliver features faster by helping you to define resources on any cloud
+    using your favorite languages."
+form:
+  gotowebinar_key: "6999643165114589707"
+    hubspot_form_id: "ebc6064d-756c-4d04-ac63-31ad16780ab5"
```
Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md:38]:
```suggestion
  gotowebinar_key: "6999643165114589707"
  hubspot_form_id: "ebc6064d-756c-4d04-ac63-31ad16780ab5"
```

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  presenters:
+    - name: Praneet Loke
+      role: Software Engineer, Pulumi
+  learn:
+    - Set up GitLab
+    - Create Pipelines for Infrastructure Updates
+    - Define an Infrastructure Code Review Process
+    - Setup Pulumi
+    - Define Resources in AWS
+  duration: 02:00:00
```
Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index-md.md:29]:
```suggestion
  duration: 2 hours
```

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  Pulumi and GitLab. "
+featured: false
+pre_recorded: false
+pulumi_tv: false
+preview_image: /images/webinar/gitlab-demo.png
+unlisted: false
+gated: true
+type: webinars
+external: false
+url_slug: gitops-11-16-2020
```
Comment (zchase) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index.md:13]:
```suggestion
url_slug: workshop-gitops-with-pulumi-and-gitlab
```

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  duration: 2 hours
+  title: "Workshop: GitOps with Pulumi and GitLab"
+  description: "Deploy and update your infrastructure the same way you deliver the
+    rest of your applications: with  code delivered through your CI/CD pipeline.
+    Learn how Pulumi’s modern infrastructure as code platform helps your team
+    deliver features faster by helping you to define resources on any cloud
+    using your favorite languages."
+form:
+  gotowebinar_key: "6999643165114589707"
+  hubspot_form_id: "ebc6064d-756c-4d04-ac63-31ad16780ab5"
```
Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index.md:38]:
```suggestion
  hubspot_form_id: "018aec45-eca9-4699-bfbb-698377c67457"
```

File: content/webinars/workshop-gitops-with-pulumi-and-gitlab/index.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+url_slug: workshop-gitops-with-pulumi-and-gitlab
+hero:
+  title: "Workshop: GitOps with Pulumi and GitLab"
+  image: /images/webinar/gitlab-demo.png
+main:
+  sortable_date: 2020-11-16T13:00:00.000-05:00
+  datetime: Mon Nov 16, 2020 AT 11:00AM PT
+  presenters:
+    - name: Praneet Loke
+      role: Software Engineer, Pulumi
```
Comment (isaac-pulumi) [inline content/webinars/workshop-gitops-with-pulumi-and-gitlab/index.md:22]:
```suggestion
    - name: Praneet Loke
      role: Software Engineer, Pulumi
    - name: Paul Stack
      role: Software Engineer, Pulumi
    - name: Joe Randazzo
      role: Solutions Architect, GitLab
```

**Review by isaac-pulumi — COMMENTED** [review]
updated speakers and hubspot form

**Review by zchase — APPROVED** [review]
👍

**Pull request merged**